### DISCUSSION TITLE
ML task report issue

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Ed/Community#566] [Bugs] [answered]
Hi Team,
We are using 4.6.0 in ML
We are facing a issue in the ML task report
[696F58EB4B83DB9EE529E06A1503F89F_20230619.csv](https://github.com/Sunbird-Ed/Community/files/11949148/696F58EB4B83DB9EE529E06A1503F89F_20230619.csv)
In the above mentioned file see the column "Task Evidence".  Copy that link and try that file does not exist.
this is the issue which we are facing.

**Comment by chitranshu-keshav** [comment]
@kiranharidas187 can you please help on the above issue

**Comment by kiranharidas187** [comment]
@Venkatesh-Dayani23  In which environment is this issue happening??

cc- @aks30

**Comment by aks30** [comment, marked as answer]
@Venkatesh-Dayani23 Looks like there is a mismatch in cloud storage credentials supplied via env/config-map to **ml-core-service** (Refer - https://github.com/project-sunbird/sunbird-devops/blob/release-6.0.0/ansible/roles/stack-sunbird/templates/ml-core-service.env 
 and the value for the same most likely will be set in pvt repo at path - ansible/inventory/{{env}}/managed-learn/common.yml) and the ones set in **config.ini for ml-analytics-server** (Refer - https://github.com/project-sunbird/sunbird-devops/blob/4eef8d0703457ee79563d6a44226fe3cf65f1957/ansible/roles/ml-analytics-service/defaults/main.yml#L20C1-L20C31 and https://github.com/project-sunbird/sunbird-devops/blob/4eef8d0703457ee79563d6a44226fe3cf65f1957/ansible/roles/ml-analytics-service/templates/config.j2#L33 again set in pvt repo at path - ansible/inventory/{{env}}/managed-learn/common.yml)

Ideally both should be same and the bucket should be public with ACL policy set to accessible only via complete link

Cc - @vijiurs @Ashwiniev95 @rakeshSgr @adithyadinesh0412
